install.js

```diff
@@ -337,3 +337,9 @@ function Deferred() {
   }.bind(this));
   Object.freeze(this);
 }
+
+
+if (require.main === module) {
```

Comment:
`require.main` could be unavailable if using esm. See https://nodejs.org/api/esm.html#esm_differences_between_es_modules_and_commonjs and https://stackoverflow.com/a/60309682/2723305.